js/bitfinex2.js

```diff
@@ -1491,7 +1491,7 @@ module.exports = class bitfinex2 extends bitfinex {
         let method = 'privatePostAuthRMovementsHist';
         if (code !== undefined) {
             currency = this.currency (code);
-            request['currency'] = currency['id'];
+            request['currency'] = currency['code'].toUpper ();
```

Comment:
This is a bit wrong – we should never send unified CCXT codes and CCXT symbols to the exchange as is.
Also, the code of the currency must always be uppercase, if that's not the case, it has to be fixed as well.
We send out exchange-specific ids, and we receive exchange-specific ids, so, it should be `currency['id']` here. We use unified symbols internally. Also, it does not make much sense to uppercase it every single time, instead I suggest adding `uppercaseId` to the currency structure once upon parsing it, if that is absolutely necessary.